### DISCUSSION TITLE
Sheet component not closed on web by pressing Esc (on web)

### DISCUSSION THREAD

**Discussion by NavyAnt24** [tamagui/tamagui#1539] [General]
### Current Behavior

On web, when pressing Esc, sheet component stays open.

### Expected Behavior

I would expect the sheet to close when pressing escape.

### Tamagui Version

```markdown
1.48.11
```

### Reproduction

```markdown
-
```

**Comment by EliRobinson** [comment]
+1 with this request. If this is a form of a "Modal" (which I believe it seems to be), based on WAI recommendations (requirements?) modals should allow closing with escape.

https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/#kbd_label
